refs/schemas/priv/RFC-PRIV.md

```diff
@@ -476,6 +479,59 @@ It may happen that two or more Retention Policies conflict, e.g. the same data m
 
 In such cases, `NO-LESS-THAN` takes priority over `NO-LONGER-THAN`. The data is kept.
 
+## Lifecycle of Legal Bases for data processing
+
+Each System SHOULD have a knowledge (i.e. a configuration) of a set of [Legal Base](#legal-base) objects.
+Those objects define, for that particular System, what makes it legal to process particular data categories in a particular way for a particular purpose.
+For example, a System might be configured to store and use (`processing-category`:{`STORING`, `USING`}) `CONTACT` data under the `CONTRACT` Legal Base because it needs the data in order to perform a service for the Data Subject.
+The same System might be configured to share (`processing-category`:`SHARING`) `CONTACT` data only under `CONSENT` Legal Base, i.e. if the Data Subject explicitly approves it.
+
+Once the System knows the Legal Bases for each [Privacy Scope](#privacy-scope), it relies on [Legal Base Events](#legal-base-event) and [Consents](#consents) to determine, for each user, the active Legal Bases. E.g. when a Data Subject starts using the System services (`SERVICE-START` type of event) the Privacy Scope associated with `CONTRACT` Legal Base becomes active.
+
+For more details, refer to [Expected Behavior of Implementing Systems](./expected-behavior.md).
+
+### Legal Base
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `legal-base` | 1-* | One or more [Legal Base Term(s)](#legal-bases)|
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) covered by the given Legal Base(s). In absence of any indication the Event is considered to concern all the possible scope. |
+
+
+### Legal Base Event
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | Data Subject concerned by the event, identified by one or more of their [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+| `event-type` | 1 | Any of the [Event Terms](#events) |
+| `legal-base` | 1-* | Any [Legal Base Term](#legal-bases) other than `CONSENT`|
```

Comment:
One Event should correspond to a single legal base - start of contract, consent given, consent revoked

Reply:
I agree the conceptual model of an Event can have multiple legal bases.
But since there is also a property `data-reference` (and might be some more in the future), the cardinality `1` makes more sense in order to know for which legal base a property belongs to (e.g. data reference to a contract).

Reply:
Not necessairly. Imagine I am a lawyer, and I use `data-reference` for a particular case (while my client has several cases I work on). I can still associate this particular data reference with several legal bases?